ansible/roles-infra/infra-osp-template-generate/templates/nested_version/cloud_template_master.j2

```diff
@@ -25,7 +25,9 @@ resources:
       user: { get_resource: {{ guid }}-project_user }
       roles:
         - {project: {{ osp_project_name }}, role: _member_ }
+{% if osp_use_swift | d(true) | bool %}        
```

Comment:
I think we can consider getting rid of this file. @fridim left it here when we first transitioned to the single one last year, but I think the disconnected config is the only thing that ever used nested. And that is being converted right now.

Reply:
I just added that variable because I need that to not use SWIFT when I deploy 4.4 (on Green). But I'm happy to remove the nested config(s) when the time comes.